utils/redis_helper.py

```diff
@@ -0,0 +1,52 @@
+from django.conf import settings
+from utils.redis_client import RedisClient
+from utils.redis_serializers import DjangoModelSerializer
+
+
+class RedisHelper:
+
+    @classmethod
+    def _load_objects_to_cache(cls, key, objects):
+        conn = RedisClient.get_connection()
+
+        serialized_list = []
+        for obj in objects:
+            serialized_data = DjangoModelSerializer.serialize(obj)
+            serialized_list.append(serialized_data)
+
+        if serialized_list:
+            # *serialized_list means all data in this list
+            conn.rpush(key, *serialized_list)
+            conn.expire(key, settings.REDIS_KEY_EXPIRE_TIME)
+
+
+    @classmethod
+    def load_objects(cls, key, queryset):
```

Comment:
if key exists, get the list from the redis. If not, get the list from the queryset and load it into the redis